instructor/templates/instructor/assignment_sidebar.html

```diff
@@ -147,6 +147,16 @@
                 Sample Prep
             </a>
         </div>
+        <div role="link" class='scb_ab_s_sidebar_page_link
+            {% if page_name == "facs_analyze" %}scb_ab_s_sidebar_page_name_selected{% endif %}'>
+            <a class="scb_s_assignment_sidebar_link {% if last_page_number < 12 %} disabled {% endif %}"
```

Comment:
Anna, here you are assuming I have any knowledge of any kind on the number of pages for this project: this assumption is wrong.
If I need to acquire such knowledge before reviewing this PR, please point me to the right issue #, documentation or code.
cc @pdpinch